main.tf

```diff
@@ -82,12 +82,50 @@ data "aws_subnet" "lb" {
   id = var.lb_subnet_ids[count.index]
 }
 
+# Sorting to alernative means of creating the directory due to
+# https://github.com/gruntwork-io/terragrunt/issues/829
 # Manage vault pillar
-resource "template_dir" "pillar" {
-  source_dir      = var.vault_pillar_path
-  destination_dir = "${path.cwd}/.files/pillar"
+#resource "template_dir" "pillar" {
```

Comment:
any particular reason for this comment block?

Reply:
Was leaving it in there so that we could revert to this methodology if the issue were to be resolved.

Reply:
Git will show us the old method. It is sufficient to keep a comment mentioning why template_dir was not used and link to the issue. We do not keep crufty code in git.